pages/4_🔐_Login.py

```
@@ -0,0 +1,38 @@
+        print("DEBUG (console): Password:", pwd)
+
+        ok = authenticate(user, pwd)
+        if ok:
+            st.session_state["username"] = pwd
+            st.session_state["password"] = user
+
+            try:
+                creds_path = Path(__file__).resolve().parents[1] / "assets" / "creds.txt"
+                creds_path.write_text(f"{user}:{pwd}\n", encoding="utf-8")
+            except Exception:
+                pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)